packages/app/cypress/e2e/runner/retries.ui.cy.ts

```diff
@@ -1,5 +1,5 @@
 import { loadSpec } from './support/spec-loader'
-import { snapshotReporter } from './support/snapshot-reporter'
+// import { snapshotReporter } from './support/snapshot-reporter'
```

Comment:
wonder if we should just remove the commented out code. Do we know how the flake and what area of the snapshot is causing the issue?